### PR TITLE
Remove note about Azure KeyVault-based secret scopes and service principals

### PR DESCRIPTION
**Pull request by alexott**
## Tests

- [x] `make test` run locally
- [x] relevant change in `docs/` folder
- [x] covered with integration tests in `internal/acceptance`
- [x] relevant acceptance tests are passing
- [ ] using Go SDK

### CODE REVIEW

**Comment by alexott** [comment]
`azure-prod` acceptance test passed

File: internal/acceptance/secret_scope_test.go [hunk 1]
```diff
@@ -58,3 +59,47 @@ func TestAccSecretScopeResource(t *testing.T) {
 		),
 	})
 }
+
+func TestAccSecretScopeResourceAkvWithSp(t *testing.T) {
+	cloudEnv := os.Getenv("CLOUD_ENV")
```
Comment (nfx) [inline internal/acceptance/secret_scope_test.go:64]: `CLOUD_ENV` is checked automatically

File: internal/acceptance/secret_scope_test.go [hunk 2]
```diff
@@ -58,3 +59,47 @@ func TestAccSecretScopeResource(t *testing.T) {
...
+	keVaultResourceID := os.Getenv("TEST_KEY_VAULT_RESOURCE_ID")
+	if cloudEnv != "azure" || keVaultResourceID == "" || os.Getenv("") == "ARM_CLIENT_ID" {
+		t.Skipf("not available on %s or don't have key vault and service principal defined", cloudEnv)
+	}
+
+	scope := qa.RandomName("tf-")
+	workspaceLevel(t, step{
+		Template: fmt.Sprintf(`
```
Comment (nfx) [inline internal/acceptance/secret_scope_test.go:72]:
```
Template: `
resource "databricks_secret_scope" "my_scope" {
			name = "akv-{var.RANDOM}"
			keyvault_metadata {
				resource_id = "{env.TEST_KEY_VAULT_RESOURCE_ID}"
				dns_name    = "{env.TEST_KEY_VAULT_DNS_NAME}"
			}
		}
`
```

**Review by nfx — CHANGES_REQUESTED** [review]

File: internal/acceptance/secret_scope_test.go (hunk 2 above)
Comment (alexott) [reply internal/acceptance/secret_scope_test.go:72]: Hmmm, If I remove `resourceCheck` then it fails with on non-empty plan check...

**Review by nfx — APPROVED** [review]

**Pull request merged**